src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -223,3 +223,101 @@ The span finishes once the operation has been executed. The span `status` is set
 When the operation throws an `Exception`, Sentry's SDK associates this exception to the running span. If you haven't set the SDK to swallow the exception and capture it, the span and SentryEvent will be linked when viewing it on the **Issue Details** page in sentry.io.
 
 For more information see our [file I/O integration](/platforms/android/configuration/integrations/file-io/).
+
+### User Interaction Instrumentation
+
+The UI instrumentation, once enabled, captures transactions for a set of different user interactions, which include clicks, scrolls and swipes. The SDK composes the Transaction name out of the host Activity and the `id` of the view, which captured the user interaction; for example, `LoginActivity.login_button`. The Transaction operation is set to `ui.action` plus the interaction type (one of `click`, `scroll` or `swipe`).
+
+<Note>
+
+If the view does not have the `id` assigned, the transaction will not be captured, because it cannot be uniquely identified otherwise.
+
+</Note>
+
+_The UI instrumentation is disabled by default, but you may enable it by setting:_
+
+```xml {filename:AndroidManifest.xml}
+<manifest>
+    <meta-data android:name="io.sentry.traces.user-interaction.enable" android:value="true" />
+</manifest>
+```
+
+The transaction finishes automatically after the specified [idleTimeout](/platforms/android/configuration/options/#idle-timeout) and all of its child spans are finished. The `idleTimeoout` defaults to 3 seconds; you can also disable the idle timeout by setting it to `null` - the transaction must be finished manually in this case.
+
+<Note>
+
+If the UI transaction has idled, but didn't have any child spans added, it will be dropped.
+
+</Note>
+
+To finish the transaction manually, you can do:
+
+```java
+import io.sentry.Sentry;
+
+ISpan span = Sentry.getSpan();
+if (span != null) {
+  span.finish();
+}
+```
+
+```kotlin
+import io.sentry.Sentry
+
+val span = Sentry.getSpan()
+span?.finish()
+```
+
+In case the host Activity transitions into a non-interactive state (e.g. `onPause`), the UI transaction will be scheduled to finish automatically (as soon as all of its child spans are finished).
+
+Transactions are always bound to the Scope automatically if there's none set. Because of that, you may create spans using manual instrumentation, and those spans will be automatically associated with the running UI transaction.
```

Comment:
```suggestion
Transactions are always bound to the scope automatically if there's none set. Because of that, you can create spans using manual instrumentation, and those spans will be automatically associated with the running UI transaction.
```